biz.aQute.bndlib/src/aQute/bnd/osgi/resource/ResourceUtils.java

```diff
@@ -546,6 +546,8 @@ public static VersionedClause toVersionClause(Resource resource, String mask) {
 		if (resource.getCapabilities(WORKSPACE_NAMESPACE).isEmpty()) {
 			Macro macro = new Macro(new Processor());
 			Version version = getVersion(idCap);
+			if (version == null)
+				version = Version.emptyVersion;
```

Comment:
It's a common practice everywhere else in OSGi (missing Bundle-Version in manifest, missing version attrib on export, etc, all imply 0.0.0).